res/layout/subtitles_wizard_item_file.xml

```diff
@@ -2,18 +2,17 @@
                   android:id="@+id/file_container"
                   android:layout_width="match_parent"
                   android:layout_height="wrap_content"
-                  android:layout_marginLeft="@dimen/content_list_left_margin"
                   android:gravity="center_vertical"
                   android:orientation="horizontal" >
 
         <ImageView android:id="@+id/file_icon"
-	               android:layout_width="@dimen/video_list_thumbnail_width"
-	               android:layout_height="@dimen/video_list_thumbnail_height"
+	               android:layout_width="54dip"
+	               android:layout_height="54dip"
 	               android:scaleType="center"
-	               android:layout_marginTop="@dimen/video_list_thumbnail_margin_top"
-	               android:layout_marginBottom="@dimen/video_list_thumbnail_margin_bottom"
-	               android:layout_marginLeft="@dimen/video_list_thumbnail_margin_left"
-	               android:layout_marginRight="@dimen/content_list_space_between_thumbnail_and_name"/>
+	               android:layout_marginTop="3dip"
```

Comment:
I see that you are replacing abstracted dimensions by hard coded ones. Should it be added into dimens.xml?
Perhaps not, I am open on the topic.